draft-packages/select/KaizenDraft/Select/Select.tsx

```diff
@@ -108,6 +108,7 @@ export const Select = React.forwardRef<any, SelectProps>((props, ref) => {
           IndicatorsContainer,
           ClearIndicator,
           IndicatorSeparator: null,
+          LoadingMessage,
```

Comment:
I imagine `LoadingMessage` is probably exclusive to the async version so probably isn't needed here

Reply:
My assumption is that if there's a big list of data it may not instantaneously find the results from a search? Ultimately its just updating the primitive that's already baked into react-select so I figured it was safer to leave it in

Reply:
Nah it's all preloaded when it's not async :) But that's okay, if TS isn't complaining (although we aren't using the generic (which we probably should, but later), so not sure if that's the cause) then we can just leave it in until we make it tighter. If it doesn't do anything it's just useless code, but nothing will break. We can dig into it properly later.

Reply:
gotcha, just took a deeper look and your completely right :)  Following YAGNI so its gone now